internal/service/advancedclustertpf/resource_schema.go

```diff
@@ -2,9374 +2,617 @@ package advancedclustertpf
-											},
-											"ebs_volume_type": schema.StringAttribute{
-												Optional:            true,
-												Computed:            true,
-												Description:         "Type of storage you want to attach to your AWS-provisioned cluster.\n\n- `STANDARD` volume types can't exceed the default input/output operations per second (IOPS) rate for the selected volume size. \n\n- `PROVISIONED` volume types must fall within the allowable IOPS range for the selected volume size. You must set this value to (`PROVISIONED`) for NVMe clusters.",
-												MarkdownDescription: "Type of storage you want to attach to your AWS-provisioned cluster.\n\n- `STANDARD` volume types can't exceed the default input/output operations per second (IOPS) rate for the selected volume size. \n\n- `PROVISIONED` volume types must fall within the allowable IOPS range for the selected volume size. You must set this value to (`PROVISIONED`) for NVMe clusters.",
-												Default:             stringdefault.StaticString("STANDARD"),
-											},
-											"instance_size": schema.StringAttribute{
-												Optional:            true,
-												Computed:            true,
-												Description:         "Hardware specification for the instances in this M0/M2/M5 tier cluster.",
-												MarkdownDescription: "Hardware specification for the instances in this M0/M2/M5 tier cluster.",
-											},
-											"node_count": schema.Int64Attribute{
-												Optional:            true,
-												Computed:            true,
-												Description:         "Number of nodes of the given type for MongoDB Cloud to deploy to the region.",
-												MarkdownDescription: "Number of nodes of the given type for MongoDB Cloud to deploy to the region.",
-											},
-										},
-										CustomType: ElectableSpecsType{
-											ObjectType: types.ObjectType{
-												AttrTypes: ElectableSpecsValue{}.AttributeTypes(ctx),
-											},
-										},
-										Optional:            true,
-										Computed:            true,
-										Description:         "Hardware specifications for all electable nodes deployed in the region. Electable nodes can become the primary and can enable local reads. If you don't specify this option, MongoDB Cloud deploys no electable nodes to the region.",
-										MarkdownDescription: "Hardware specifications for all electable nodes deployed in the region. Electable nodes can become the primary and can enable local reads. If you don't specify this option, MongoDB Cloud deploys no electable nodes to the region.",
-									},
+									"electable_specs": specsAttribute(ctx, "Hardware specifications for all electable nodes deployed in the region. Electable nodes can become the primary and can enable local reads. If you don't specify this option, MongoDB Cloud deploys no electable nodes to the region."),
```

Comment:
I guess the specsAttribute was defined manually?

Reply:
it was first created automatically, but then I extracted that to a function as multiple attributes were the same structure